health/REFERENCE.md

```diff
@@ -394,21 +394,21 @@ By defining labels inside of `netdata.conf`, you can now apply labels to alarms.
 line to any alarms you'd like to apply to hosts that have the label `room = server`.
 
 ```yaml
-label: room = server
+host lables: room = server
 ```
 
 You can also combine labels when applying them to alarms. For example, if you want to raise a specific alarm only for hosts 
 inside a room that were installed at a specific time, you can write the following label line:
```

Comment:
You can't do that with a simple pattern.

label -> host labels

Reply:
You forgot about simple patterns.

Reply:
I am doing references to simple pattern on line 411, they idea behind these changes on our manual was to increase complexity until the last example, this is the motive it was not cited before. I also wanted to avoid repetition of ideas.